### ISSUE TITLE
[BUG] [1.5.1] Cannot read properties of undefined (reading 'id')

### ISSUE THREAD

**Issue by Ealrann** [Chia-Network/chia-blockchain-gui#897]
### What happened?

Hello,

The NFT page always crash and proposed me to open a bug issue on github.

I'm on branch:
Chia : release/1.5.1 (commit a041576c12c23b03d665c9ade775ee3f2bfc32a5 Cherry pick)
Chia-GUI : release/1.5.1 (commit 176d522c8da3a1212535c77296a9210b48e13d87  Header sync state)

### Version

1.5.1.dev

### What platform are you using?

Linux

### What ui mode are you using?

GUI

### Relevant log output or stacktrace

```shell
Cannot read properties of undefined (reading 'id')

URL
#/dashboard/nfts

Stacktrace
at webpack://@chia/gui/src/components/nfts/NFTProfileDropdown.tsx:84:31 useMemo(
at webpack://@chia/gui/src/components/nfts/NFTProfileDropdown.tsx:82:45 useMemo(
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:168:376 $h
at webpack://@chia/gui/node_modules/react/cjs/react.production.min.js:23:112 useLayoutEffect
at webpack://@chia/gui/src/components/nfts/NFTProfileDropdown.tsx:77:30 useMemo(
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:157:136 Ch
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:180:153 oi
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:269:342 a=b.pendingProps;f=e._init;e=f(e._payload);b.type=e;f=b.tag=hk(e);a=lg(e,a);switch
at webpack://@chia/gui/node_modules/react-dom/cjs/react-dom.production.min.js:250:346 Pj
```

**Comment by Ealrann** [comment]
Looks fixed with 627706b419357c9a9d8f826c058c5c27dfb5b8f1 :tada:

**Comment by Ealrann** [comment]
No, crash still happen with the last 1.5.1, commit b2de97a8e95234046e48691b754a269de9440c4d
![Screenshot from 2022-08-13 00-04-42](https://user-images.githubusercontent.com/723000/184450654-5d13ed1a-2142-4d96-89e3-f4c839cbab4e.png)

**Comment by paninaro** [comment]
This should be fixed in 1.5.1
